locales/en-US/app.ftl

```diff
@@ -784,6 +784,21 @@ TrackPower--tooltip-energy-used-in-preview-watthour = { $value } Wh
 TrackPower--tooltip-energy-used-in-preview-milliwatthour = { $value } mWh
   .label = Energy used in the current selection
 
+## TrackCarbon
```

Comment:
Leave an empty line after the group comment. Also add a comment (for the group) explaining that `g` and `mg` are abbreviations for gram and microgram, since those are not typically used in a browser.